tests/runtest.sh

```diff
@@ -368,7 +368,7 @@ function create_core_overlay {
     if [ -d "$mscorlibDir/bin" ]; then
         cp -f -v "$mscorlibDir/bin/"* "$coreOverlayDir/" 2>/dev/null
     fi
-    cp -n -v "$testDependenciesDir"/* "$coreOverlayDir/" 2>/dev/null
+    cp -f -v "$testDependenciesDir"/* "$coreOverlayDir/" 2>/dev/null
```

Comment:
From a cursory look, it does look like that's where we'd want the test binaries from, but it looks like there maybe something else in that directory that is causing huge numbers of tests to fail in the CI.